lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -760,6 +765,107 @@ public void testQueryNotSet()
         accessControlManager.checkCanKillQueryOwnedBy(new SystemSecurityContext(bob, queryId), "any");
     }
 
+    @Test
+    public void testCatalogRulesDocsExample()
+    {
+        String rulesFile = new File("../../docs/src/main/sphinx/security/catalog_rules.json").getAbsolutePath();
+        SystemAccessControl accessControlManager = newFileBasedSystemAccessControl(ImmutableMap.of("security.config-file", rulesFile));
+
+        // admin to access the mysql and the system catalog
+        accessControlManager.checkCanAccessCatalog(new SystemSecurityContext(admin, queryId), "mysql");
+        accessControlManager.checkCanAccessCatalog(new SystemSecurityContext(admin, queryId), "system");
+        accessControlManager.checkCanAccessCatalog(new SystemSecurityContext(admin, queryId), "hive");
```

Comment:
nit: I'd recommend to add all the hive checks together at the end - it's easier to see that way that `hive` is allowed to all identities.